viewer/packages/cad-geometry-loaders/src/sector/culling/TakenSectorTree.test.ts

```diff
@@ -5,27 +5,33 @@
 import * as THREE from 'three';
 import { generateSectorTree, expectContainsSectorsWithLevelOfDetail, Mutable } from '../../../../../test-utilities';
 import { PropType } from '../../utilities/reflection';
-import { SectorMetadata, CadModelMetadata, LevelOfDetail } from '@reveal/cad-parsers';
+import {
+  SectorMetadata,
+  CadModelMetadata,
+  LevelOfDetail,
+  BaseSectorMetadata,
+  V8SectorMetadata
+} from '@reveal/cad-parsers';
 import { traverseDepthFirst } from '@reveal/utilities';
 import { DetermineSectorCostDelegate, PrioritizedWantedSector } from './types';
 import { TakenSectorTree } from './TakenSectorTree';
 
-type FacesFile = PropType<SectorMetadata, 'facesFile'>;
+type FacesFile = PropType<BaseSectorMetadata & V8SectorMetadata, 'facesFile'>;
```

Comment:
```suggestion
type FacesFile = PropType<V8SectorMetadata, 'facesFile'>;
```

You wouldn't need the full type here, `V8SectorMetadata` will do (?)